src/Symfony/Component/HttpKernel/Attribute/Serialize.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\HttpKernel\Attribute;
+
+/**
+ * Controller tag to serialize response.
+ */
+#[\Attribute(\Attribute::TARGET_METHOD)]
+class Serialize
```

Comment:
this attribute must be final, as the logic in the listener does not account for child classes

Reply:
done